src/cpp/database/database.cpp

```diff
@@ -3159,6 +3167,42 @@ DatabaseDump Database::dump_data_(
     return data_dump;
 }
 
+void Database::clear_database()
```

Comment:
I'd use a name referring to the `data`, so that it is clear that is only the `data` part that is cleared. For example: `clear_statistics_data()`.